x-pack/plugins/session_view/public/components/session_view/styles.ts

```diff
@@ -26,9 +26,22 @@ export const useStyles = ({ height = 500 }: StylesDeps) => {
       height: `${height}px`,
     };
 
+    const searchBar: CSSObject = {
+      position: 'relative',
+      marginLeft: `${euiTheme.size.xs} !important`,
+      marginRight: `${euiTheme.size.xs} !important`,
+    };
+
+    const buttonsEyeDetail: CSSObject = {
```

Comment:
could it be:

```js
const buttonsEyeDetail: CSSObject = {
      margin: `${euiTheme.size.xs} !important`
};
```

Reply:
you could also do
```
margin: `${euiTheme.size.m} ${euiTheme.size.xs} !important`
```
if you want keep different margins between top/bottom and left/right